.github/workflows/benchmark_pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Benchmark PR
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    branches: [main]
+
+jobs:
+  benchmark-pr:
+    name: Benchmark PR
+    if: contains(github.event.pull_request.labels.*.name, 'PR talk')
```

Comment:
Let's have a dedicated label, "benchmark PR" or something? Would make it more intuitive esp. for future contributors.

Reply:
Good idea!